user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementMain.java

```diff
@@ -530,8 +531,37 @@ private void openRegistrationDialog() {
     var registrationUseCase = new MeasurementUpload(asyncService, context,
         ConverterRegistry.converterFor(
             MeasurementRegistrationInformationNGS.class), messageSourceNotificationFactory);
-    DialogBody.with(dialog, registrationUseCase, registrationUseCase);
+    //DialogBody.with(dialog, registrationUseCase, registrationUseCase);
+    var templateComponent = new MeasurementTemplateSelectionComponent(Map.of(Domain.Genomics, new WorkbookDownloadStreamProvider() {
+      @Override
+      public String getFilename() {
+        return FileNameFormatter.formatWithVersion("ngs_measurement_registration_sheet", 1, "xlsx");
+      }
+
+      @Override
+      public Workbook getWorkbook() {
+        return NGSWorkbooks.createRegistrationWorkbook();
+      }
+    }, Domain.Proteomics, new WorkbookDownloadStreamProvider() {
+      @Override
+      public String getFilename() {
+        return FileNameFormatter.formatWithVersion("pxp_measurement_registration_sheet", 1, "xlsx");
+      }
+
+      @Override
+      public Workbook getWorkbook() {
+        return ProteomicsWorkbooks.createRegistrationWorkbook();
+      }
+    }));
+    templateComponent.addDomainSelectionListener(event -> {
+      var selectedDomain = templateComponent.selectedDomain();
+      log.info("Selected new domain: " +  selectedDomain);
```

Comment:
```suggestion
      log.debug("Selected new domain: " +  selectedDomain);
```
Probably debug level information.